src/api/app/models/event/upload.rb

```diff
@@ -12,33 +12,21 @@ def self.message_bus_routing_key
 
 # == Schema Information
 #
-# Table name: packages
+# Table name: events
```

Comment:
I think you need to rebase... these changes in the annotation are about the refactoring @Ana06 did to the Events module.

Reply:
I have already rebased. It looks like Ana made the changes but didn't commit the changes that annotate made so I'm committing them now.

Reply:
I had already done this 🤔

Reply:
@Ana06 it looks like you might've forgotten to commit some of the event models? For example `app/models/event/upload.rb` still has the wrong table name in the annotate comments:

https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/event/upload.rb

Reply:
yes... 🙈